pagopa/io-app
# feat(Carta Giovani Nazionale): [#179282600] Integrates CGN Merchant Landing page discount option

## Short description
Implements the webview to properly handle the landing page discount type.

## How to test
Update `io-dev-api-server` to latest commit to ensure to support this feature


https://user-images.githubusercontent.com/3959405/130094043-6944ca7f-79de-497b-ab44-bf014060cb74.mov
